README.md

```diff
@@ -1 +1,22 @@
-# device-images
\ No newline at end of file
+OpenBMC power supply firmware
+
+This directory contains P11 power supply firmware images supplied by the
+vendors. Each PSU firmware should reside in a directory named as the PSU CCIN.
+The firmware image file prefix should be replaced with IBM PSU CCIN and postfix
```

Comment:
I would remove the "IBM" text, so other companies can use the same naming convention.

Reply:
I will remove IBM, should I replace CCIN with model number?

Reply:
Not sure that it matters, whatever it's called in inventory manager is fine